admin/app/controllers/solidus_admin/base_controller.rb

```diff
@@ -1,7 +1,11 @@
 # frozen_string_literal: true
 
 module SolidusAdmin
-  class BaseController < Spree::BaseController
+  class BaseController < ApplicationController
+    include ActiveStorage::SetCurrent
+    include ::SolidusAdmin::Auth
+    include Spree::Core::ControllerHelpers::Store
```

Comment:
I understand they'll be used, but are they needed in the context of the PR configuring authentication? :thinking:

Reply:
_Been a long time, but I'll add an answer anyway 😊._

Those are not unused, `current_store` is necessary for the sidebar, while ActiveStorage::SetCurrent is needed in order to generate blob urls, removing it would cause this error (I erroneously thought it was only needed for uploading):

<img width="1162" alt="image" src="https://github.com/solidusio/solidus/assets/1051/60a7c5d7-9aee-497f-bee3-0d74e1aef28e">

See https://github.com/rails/rails/blob/7-0-stable/activestorage/app/controllers/concerns/active_storage/set_current.rb#L3-L6